ui/src/org.ts

```diff
@@ -21,24 +30,33 @@ export enum RegistrationFlowState {
 const orgStore = remote.createStore<Org>();
 export const org = orgStore.readable;
 
-// Api 
-export const getOrg = (id: string): Promise<Org> => api.get<Org>(`orgs/${id}`)
+const orgProjectsStore = remote.createStore<OrgProjects>();
+export const orgs = orgProjectsStore.readable;
```

Comment:
```suggestion
export const projects = projectsStore.readable;
```

Otherwise this is miss-leading.